metroae

```diff
@@ -1285,7 +1285,7 @@ function docker_exec_interactive {
         environment=" -e METROAE_PASSWORD=$METROAE_PASSWORD"
     fi
     if [[ ! -z $SSHPASS_PASSWORD ]]; then
-        sshpass="sshpass -p$SSHPASS_PASSWORD"
+        sshpass="sshpass -p '$SSHPASS_PASSWORD'"
```

Comment:
No space after -p (I think it matters...)

Reply:
done